.github/workflows/compile.yml

```diff
@@ -12,8 +12,8 @@ jobs:
     strategy:
       matrix:
         go-version:
-           - 1.13.x
            - 1.14.x
+           - 1.15.x
```

Comment:
Wondering if we should also keep `1.13.x` around just so we're informed if we introduce a breaking change for an older version of Go. Thoughts?

Reply:
If we did and the build failed, would that prevent us from merging the PR?

Reply:
Yeah, but likely won't run into that problem. And when we do, we can make a decision on whether or not to remove that version of the build matrix, or update the code in question, or create a separate parallel job that runs on Go 1.13 that's allowed to fail. 

Also totally fine keeping things as they are in this PR. I'm more so thinking about making sure we stay informed when/if we introduce something messes with things.

Reply:
Okay, I've added 1.13 back in.  You make a good point.